consensus/api/proto/consensus_common.proto

```diff
@@ -56,14 +56,16 @@ enum ProposeTxResult {
     UnsortedKeyImages = 26;
     ContainsSpentKeyImage = 27;
     DuplicateKeyImages = 28;
-    MissingTxOutMembershipProof = 29;
-    InvalidTxOutMembershipProof = 30;
-    InvalidRistrettoPublicKey = 31;
-    InvalidLedgerContext = 32;
-    Ledger = 33;
-    MembershipProofValidationError = 34;
-    TxFeeError = 35;
-    KeyError = 36;
+    DuplicateOutputPublicKey = 29;
```

Comment:
This is technically a breaking change, but right now we are in control of all client implementations so this isn't gonna negatively affect anything.

Reply:
It's worth noting that any breaking change means server upgrades are not strictly backwards compatible w.r.t. SDKs and will require an SDK update. Is there a reason to not just add the 2 new errors at the end of the enum?

Reply:
Does the SDK currently match against any of these?
My preference was to use the opportunity that none of that stuff is public yet to maintain a 1:1 mapping between this and the Rust code.